src/fides/api/ops/service/connectors/saas_connector.py

```diff
@@ -404,6 +404,35 @@ def mask_data(
         self.unset_connector_state()
         return rows_updated
 
+    def run_consent_request(
+        self,
+        node: TraversalNode,
+        policy: Policy,
+        privacy_request: PrivacyRequest,
+        identity_data: Dict[str, Any],
+        executable_preferences: List[Consent],
+    ) -> bool:
+        """Execute a consent request. Return whether the consent request to the third party succeeded.
+
+        Executable_preferences have already been filtered to just consent preferences the customer has deemed executable.
+
+        Return True if 200 OK
+        """
+        logger.info(
+            "Mocking consent request - actual logic should go here for saas connectors",
+            node.address.value,
+        )
+
+        logger.info(
+            "Demo only! Testing available consent params! identity_data: {}, executable_preferences: {}",
+            identity_data,
+            [
+                {"use": preference.data_use, "opt_in": preference.opt_in}
+                for preference in executable_preferences
+            ],
+        )
+        return True
+
```

Comment:
Currently I am adding a `run_consent_request` method to the SaasConnector. This is where we'd make requests to third-party endpoints to propagate consent.  Extra logging is added here to demonstrate what is available but should be removed before release.